### PR TITLE
Fix empty returns section

### LINKED ISSUE #9932: Napoleon emits empty "returns" sections
### Describe the bug

image

```python
def get_task(task_id):
    """
    Get a task object

    Args:
        task_id: numeric ID of task

    Returns:
        Task:
    """
```

### How to Reproduce

See above

### Expected behavior

Don't emit the empty section

### Your project

(private)

### Screenshots

_No response_

### OS

Mac, Linux/Container

### Python version

3.10

### Sphinx version

latest

### Sphinx extensions

autodoc, intersphinx, todo, viewcode, napoleon, sphinx_inline_tabs, prettyspecialmethods

### Extra tools

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Gobot1234**
### Feature or Bugfix

Bugfix

Closes #9932

### CODE REVIEW

File: tests/test_ext_napoleon_docstring.py [hunk 1]
```diff
@@ -1118,7 +1130,7 @@ def test_noindex(self):
 .. method:: func(i, j)
    :noindex:
 
-   
+
```
Comment (tk0miya) [inline tests/test_ext_napoleon_docstring.py]: I guess this change causes an error in CI. Is this an intended change? Please check it.

Comment (Gobot1234) [reply tests/test_ext_napoleon_docstring.py]: Oh yeah must have been the line trimming

Comment (Gobot1234) [reply tests/test_ext_napoleon_docstring.py]:
```suggestion
   
```

**Comment by tk0miya** [comment]
Thank you for quick work. Merging.

**Pull request merged**